### ISSUE TITLE
nsqadmin UI shows INTERNAL ERROR

### ISSUE THREAD

**Issue by Dieterbe** [bitly/nsq#623]
i was just following http://nsq.io/overview/quick_start.html with the latest nsq installed from master.

all pages have an error like the below.
![nsq-error](https://cloud.githubusercontent.com/assets/20774/9134502/f64e3936-3d04-11e5-917a-23fc1ecc2dc4.png)

**Comment by Dieterbe** [comment]
also, not sure if this is related, but the counter page looks like this:
![nsq-error2](https://cloud.githubusercontent.com/assets/20774/9134533/3550796e-3d05-11e5-9993-3e81e01feeaf.png)

**Comment by Dieterbe** [comment]
nsqadmin is throwing a lot of errors like
```
[nsqadmin] 2015/08/07 13:07:54.510965 ERROR: panic in HTTP handler - unknown response type struct { NewMessages int64 "json:\"new_messages\""; TotalMessages int64 "json:\"total_messages\""; ID string "json:\"id\"" }
```

**Comment by mreiferson** [comment]
Probably something we broke in #612 that will be fixed when #323 lands (soon)
